Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.Networking.sb.in

```diff
@@ -733,6 +736,7 @@
         SYS_dup
         SYS_dup2
         SYS_exit
+        SYS_faccessat
```

Comment:
Is this syscall required?

Reply:
Yes. All of these were added because of sandboxing errors (not speculative).